SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1441 @@
+	if ($doComparison -eq $false) {
+		$htmlOutput = $htmlOutput + "<br><font color=#FFA500>-Common password policy file not provided, Comparison has not been evaluated</font></p></H4>"
+	} else {
+		$htmlOutput = $htmlOutput + "</p></H4>"
+	}
+	#prepare ESXi servers section"
+	$htmlOutput = $htmlOutput + '<H3>ESXi servers Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "ESXi" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdExpInDays,passwdMaxFailAttempts,passwdMinimumLengthFor1CharClass,passwdMinimumLengthFor2CharClass,passwdMinimumLengthFor3CharClass,passwdMinimumLengthFor4CharClass,passwdMinimumCharLengthForPhrase | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare vCenter Servers section"
+	$htmlOutput = $htmlOutput + '<H3>vCenter Server Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "VC" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdExpInDays,passwdNotifyEmail | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare SSO section"
+	$htmlOutput = $htmlOutput + '<H3>Single Sign On Password Policy</H3><p><font color=#6495ED>SSO FQDN: ' + $ppmVariables.ssoServerFqdn + '</font></p><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "SSO" } | select-object -ExpandProperty productAttributes | select-object -property * -ExcludeProperty FQDN,productType,driftAlarm,driftMessage | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare NSXMgr servers section"
+	$htmlOutput = $htmlOutput + '<H3>NSX Manager Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "NSXMgr" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,apiPasswdMaxFailAttempts,apiPasswdMaxFailIntervalInSec,apiPasswdUnlockIntervalInSec,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare Edge servers section"
+	$htmlOutput = $htmlOutput + '<H3>NSX Edge Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where { $_.productType -match "NSXEdge" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare WSA servers section"
```

Comment:
"# Prepare the Workspace ONE Access section`

Reply:
addressed in 99029d5ec8015e0c5c645e9abc7c3384e1b3091e